### DISCUSSION TITLE
Could not get result from torus nodes Error occurred while verifying paramsaud validation failed, token values [<firebase-project-id>] dont include validation value <firebase-project-id>

### DISCUSSION THREAD

**Discussion by qqqllppp** [Web3Auth/Web3Auth#365] [Help]
I am trying to setup a new web3auth project, but for some reason I am getting this error when creating a new account for a user via firebase custom auth and passing token to web3auth login:

> Could not get result from torus nodes Error occurred while verifying paramsaud validation failed, token values [firebase-project-id] dont include validation value firebase-project-id

I have setup a project before, but now when I do it again I run into this error :/

my setup is something like this:

```
const WEB3AUTH_CLIENT_ID = 'BDBkjySwcSOosUcI7mAhUjFdOn5LqZPDTcxWlT0hXTlL5WD6GhsZz-gCnlwAwSNgdIiOzPduPiWwXiEGZZmw_74'

const adapter = new OpenloginAdapter({
                    adapterSettings: {
                        network: 'testnet',
                        clientId: WEB3AUTH_CLIENT_ID,
                        uxMode: "redirect",
                        loginConfig: {
                            jwt: {
                                name: 'LiveThree',
                                verifier: 'LiveThree',
                                typeOfLogin: "jwt",
                                clientId: WEB3AUTH_CLIENT_ID,
                            },
                        },
                    }
                });
                
const localProvider = await web3AuthInstance.connectTo(WALLET_ADAPTERS.OPENLOGIN, {
                relogin: true,
                loginProvider: 'jwt',
                mfaLevel: 'none',
                extraLoginOptions: {
                    id_token: jwtToken,
                    domain: window.location.origin,
                    verifierIdField: "sub",
                }
            });
```

**Comment by himanshuchawla009** [comment]
@rinzler99 plz help debugging this

**Comment by himanshuchawla009** [comment]
@junhuang-ho make sure your jwt token must have these two fields:-

aud: "livethree-d1d85"
iss: "https://securetoken.google.com/livethree-d1d85"

as you have configured these validations while creating your verifier on web3auth developer dashboard.

**Reply by qqqllppp** [reply]
@himanshuchawla009  what do you mean? I currently have 2 projects in my dashboard (moonlight and livethree).

All this time while using moonlight with the config I have in my app, I have no issues. But when I created a new project and just changed the minimum config in my app, I ran into that issue.

Here is what I changed in my app:

```
1. my firebase_config (just copy paste from firebase dashboard)

2. Client ID from 'BMdhaBBKwsMr1jS7Z6rikXs_DDHZaFE1S9zIHrUZRA3J5xi-pTvN_LVm7UpeVbKmwxOk3W0T5YtoPKyrZIuWleQ'

to 

'BDBkjySwcSOosUcI7mAhUjFdOn5LqZPDTcxWlT0hXTlL5WD6GhsZz-gCnlwAwSNgdIiOzPduPiWwXiEGZZmw_74'

3. Verifier from "moonlight-mumbai" to "LiveThree" (OpenloginAdapter settings)

4. Name from "moonlight" to "LiveThree"  (OpenloginAdapter settings)

5. network from "mainnet" to "testnet"  (OpenloginAdapter settings) (as in moonlight project I was using mainnet, and since livethree's mainnet not approved yet so I use testnet for it)

so for 2-5 applies to OpenloginAdapter like so (this is how my adapter settings in code looks like when I switch between values):

const adapter = new OpenloginAdapter({
                    adapterSettings: {
                        network: "testnet",  // <----------changed here from "mainnet" (5)
                        clientId: WEB3AUTH_CLIENT_ID,  // <----------changed here (2)
                        uxMode: "popup",
                        loginConfig: {
                            jwt: {
                                name: "LiveThree", // <------------ changed here from "moonlight" (4)
                                verifier: "LiveThree", // < ------- changed here from "moonlight-mumbai" (3)
                                typeOfLogin: "jwt",
                                clientId: WEB3AUTH_CLIENT_ID,
                            },
                        },
                    }
                });
```

Changing back and forth between these values in my app's code from moonlight to livethree values causes the error.

Maybe the issue is in the verifier dashboard setup? Possible to help me check those values?

------------------
If the above is not the issue, then when you say

> make sure your jwt token must have these two fields:-
> aud: "livethree-d1d85"
> iss: "https://securetoken.google.com/livethree-d1d85"

Are you referring to my app's code? Where do I set these values? Why didn't I need to set these values for Moonlight project?

**Reply by himanshuchawla009** [reply]
<img width="1304" alt="Screenshot 2022-08-04 at 10 01 44 PM" src="https://user-images.githubusercontent.com/14052374/182866474-dd4818fa-c694-4896-9949-c479ce77882b.png">

I just checked your verifier configuration, notice that there is a space in front of `livethree-d1d85`. I guess that is causing issues. 
Either you can update this verifier to remove space or create a new verifier. @junhuang-ho

**Reply by qqqllppp** [reply]
Ah I couldn't spot that from the dashboard.

Also when I try to update it, it required gas, so I decided to delete it and make a new one. But then it said can create project with same name, which I assume is from the mainnet version of the project which I cant delete because it is still processing. Is it possible for you to help me get rid of it so I can create a new project with the same name?

![Screenshot from 2022-08-04 22-12-56](https://user-images.githubusercontent.com/96459097/182868928-91a1491f-b18b-4a27-92d3-89c90afe9659.png)

I think this needs to be deleted before I can create a new project, but I cant do it from my side.

------------------

I tried to create a new testnet project with different name but it hit an error (possibly because there is still a pending project?):
> Error: failed to create the user in the db

![Screenshot from 2022-08-04 22-22-27](https://user-images.githubusercontent.com/96459097/182871226-b5072128-4308-4aa0-a3c8-29a06975f22e.png)

**Reply by himanshuchawla009** [reply]
@tainguyentt @BboyStatix  plz help here.

**Reply by BboyStatix** [reply]
@junhuang-ho so you had a verifier called "LiveThree" and then deleted it and tried to recreate it again right? 
Since that didn't work, you tried to create with a different name "livethree" which is basically the lowercased version of the previous name right?

I believe the issue may have to do with case sensitivity. For now the temporary workaround would be to pick a different name. I will investigate further.

**Reply by qqqllppp** [reply]
yes I already picked a different name thanks.

**Reply by YZhenY** [reply]
@BboyStatix can we change the error message to be mmore intuitive, in the case of clashing names

**Comment by yashovardhan** [comment]
Hey @junhuang-ho 

Just wanted to check whether your issue was resolved here. Did the name change for verifiers help you solve this issue?

**Reply by qqqllppp** [reply]
yes using a different name worked
